contacts/urls.py

```diff
@@ -7,9 +7,11 @@
 from . import views
 
 urlpatterns = [
-    url( r'^$',auth_views.LoginView.as_view(template_name="contacts/login.html"), name="login"),
+    url(r'^$', auth_views.LoginView.as_view(template_name="contacts/login.html"), name="login"),
     url(r'^signup/$', views.signup, name='signup'),
+    url(r'^create_user/$', views.create_user, name='create_user'),
     path('add_contact/', views.add_contact, name='add_contact'),
+    path('create_account/', views.create_account, name='create_account'),
```

Comment:
What is the difference between `create_user` and `create_account` views?

Reply:
`create_account` is basically for creating new contact i rename it with `create_contact` for better understanding.